### PR TITLE
Add test to check the LimitedTable.contains function

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

**Comment by kseo** [comment]
@majecty Add tests for LimitedTable.contains function

File: network/src/limited_table.rs [hunk 1]
```diff
@@ -155,4 +155,14 @@ mod tests {
...
         assert!(t.is_none());
     }
+
+    #[test]
+    fn contains_returns_true_when_item_is_inserted() {
+        let begin: Key = 11;
+        let limit: usize = 54;
+        let mut table: LimitedTable<TestItem> = LimitedTable::new(begin, limit);
+        let t1 = table.insert(TestItem);
+        assert!(t1.is_some());
```
Comment (sgkim126) [inline network/src/limited_table.rs]: ```assert_ne!(None, t1);``` will prints better message when the test failed.

Comment (majecty) [reply network/src/limited_table.rs]: Fixed it

**Comment by kseo** [comment]
@majecty @sgkim126 Ping?

**Pull request merged**